### PR TITLE
Update MakeConst.Vsix.csproj

### LINKED ISSUE #4490: Update MakeConst.Vsix.csproj from .NET Framework net461 to LTS (or current) version
The *MakeConst.Vsix.csproj* project file targets a .NET version which is no longer supported. This is an auto-generated issue, detailed and discussed in [dotnet/docs#22271](https://github.com/dotnet/docs/issues/22271).

See line 39 in [csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj](../blob/main/csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj#L39).

| Target version | End of life | Release notes | Nearest LTS TFM version |
| --- | --- | --- | --- |
| `v4.6.1` | N/A | [v4.6.1 release notes](https://devblogs.microsoft.com/dotnet/net-framework-4-6-1-is-now-available) | `net48` |

Consider upgrading the project to either the current release, or the nearest LTS TFM version.

If this project is intentionally targeting an unsupported version, you can optionally configure to ignore this automated issue. Create a file at the root of the repository, named *dotnet-versionsweeper.json* and add an `ignore` entry following the [globbing patterns detailed here](https://docs.microsoft.com/dotnet/api/microsoft.extensions.filesystemglobbing.matcher#remarks).

```json
{
    "ignore": [
        "**/csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj"
    ]
}
```

### LINKED ISSUE #4561: Upgrade from `v4.6.1` to LTS (or current) version
The following project file(s) target a .NET version which is no longer supported. This is an auto-generated issue, detailed and discussed in [dotnet/docs#22271](https://github.com/dotnet/docs/issues/22271).

| Target version | End of life | Release notes | Nearest LTS TFM version |
| --- | --- | --- | --- |
| `v4.6.1` | N/A | [v4.6.1 release notes](https://devblogs.microsoft.com/dotnet/net-framework-4-6-1-is-now-available) | `net48` |

  - [ ] [csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj](../blob/main/csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj#L39)

Consider upgrading projects to either the current release, or the nearest LTS TFM version.

If any of these projects are intentionally targeting an unsupported version, you can optionally configure to ignore including them in this automated issue. Create a file at the root of the repository, named *dotnet-versionsweeper.json* and add an `ignore` entry following the [globbing patterns detailed here](https://docs.microsoft.com/dotnet/api/microsoft.extensions.filesystemglobbing.matcher#remarks).

```json
{
    "ignore": [
        "**/path/to/example.csproj"
    ]
}
```

### PR DESCRIPTION
**Pull request by Youssef1313**
Parallel to https://github.com/dotnet/docs/pull/23583.
Closes #4490
Closes #4561

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
I like this change as well @Youssef1313 

I'll :shipit: once we get the build status figured out.

File: csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj [hunk 1]
```diff
@@ -1,42 +1,14 @@
...
-    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
-    <UpdatePeriodically>false</UpdatePeriodically>
-    <UpdateRequired>false</UpdateRequired>
-    <MapFileExtensions>true</MapFileExtensions>
-    <ApplicationRevision>0</ApplicationRevision>
-    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
-    <IsWebBootstrapper>false</IsWebBootstrapper>
-    <UseApplicationTrust>false</UseApplicationTrust>
-    <BootstrapperEnabled>true</BootstrapperEnabled>
+    <TargetFramework>net472</TargetFramework>
```
Comment (IEvangelist) [inline csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj:6]: Can you target `net48` instead?

Comment (Youssef1313) [reply csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj:6]: I targeted net472 based on https://github.com/dotnet/docs/issues/23578#issuecomment-812013647.
@sharwell Any concerns targeting net48 for VSIX?

Comment (IEvangelist) [reply csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj:6]: I'm good with that, then let's update the [*dotnet-versionsweeper.json*](https://github.com/dotnet/samples/blob/main/dotnet-versionsweeper.json) file to ignore this. Once it is updated to `net472`, the tool would want you to upgrade to `net48`.

```json
{
    "ignore": [
        "**/csharp/roslyn-sdk/Tutorials/MakeConst/MakeConst.Vsix/MakeConst.Vsix.csproj"
    ]
}
```

**Review by sharwell — COMMENTED** [review]
LGTM assuming this project now follows the format of the Roslyn SDK analyzer template.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Comment by IEvangelist** [comment]
Close and re-open, to trigger **Snippets 5000**

**Comment by Youssef1313** [comment]
@BillWagner The issue here was never fixed so snippets5000 will always stay stuck. The reason could be that Visual Studio used by the build doesn't contain the Roslyn workload installed.

**Comment by BillWagner** [comment]
Thanks @Youssef1313

**Comment by BillWagner** [comment]
I'll go ahead and merge this, as we've pulled it down and it builds. Like cause is the lack of the roslyn workload on the container.

**Pull request merged**